metafacture-xml/src/main/java/org/metafacture/xml/SimpleXmlEncoder.java

```diff
@@ -218,6 +216,29 @@ public void setNamespaces(final Map<String, String> namespaces) {
         this.namespaces = namespaces;
     }
 
+    /**
+     * Sets the namespace(s).
+     *
+     * @param namespacesString the namespaces as a String. Key and value are separated by a '='. Multiple entries are
```

Comment:
> Key and value are separated by a '='.

This is not entirely true, the [Properties syntax](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Properties.html#load(java.io.Reader)) allows `=`, `:` and white space.